src/WaterUsePrepareRoutine.cpp

```diff
@@ -108,6 +108,10 @@ void WaterUseCalcDaily(int waterUseType, NumericMatrix dailyUse, int year, int m
 					dailyUse(0,i) = GW_day[i];
 					dailyUse(1,i) = SW_day[i] + TF_day[i];
 		}
+				break;
+
+		//no water use is considered
+		case 0: dailyUse.fill(0); // Should not happen
```

Comment:
Why do you have this case when you use an early return at the beginning of the function when the waterUseType is 0? 
Could you integrate an error if the given waterUseType is not 0,1 or 2? You can integrate it in the switch statement as default (https://www.w3schools.com/cpp/cpp_switch.asp).

Reply:
I changed it :)
I also thought that only checking the settings so deep into the code was a bit weird so i added a settings checking at the beginning of runModel.